### ISSUE TITLE
Failing with Orleans 2.0 rc1

### ISSUE THREAD

**Issue by raghur** [OrleansContrib/OrleansTestKit#36]
So I updated a .net core project using orleans 2.0 beta3 to rc1. After that my tests are failing as

```
Test Name:	should_add_item
Test FullName:	OrleansSilo.Tests.TodoGrainImplTests.should_add_item
Test Source:	D:\code\OrleansK8S\src\OrleansSilo.Tests\TodoGrainImplTests.cs : line 24
Test Outcome:	Failed
Test Duration:	0:00:03.267

Result StackTrace:	
at Orleans.TestKit.TestKitSilo..ctor()
   at OrleansSilo.Tests.TodoGrainImplTests.BeforeEach() in D:\code\OrleansK8S\src\OrleansSilo.Tests\TodoGrainImplTests.cs:line 18
Result Message:	System.TypeLoadException : Method 'GetServices' in type 'Orleans.TestKit.Streams.TestStreamProviderManager' from assembly 'OrleansTestKit, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

My `BeforeEach` looks like this:

```
    [TestFixture]
    public class TodoGrainImplTests 
    {
        private ToDoGrainImpl grain;
        private TestKitSilo silo;
        [SetUp]
        public void BeforeEach() {
            silo = new TestKitSilo();
            grain = silo.CreateGrain<ToDoGrainImpl>("RANDOM");
        }
        
        [Test]
        public async Task should_add_item()
        {
            var todo = GetTodo();
            await grain.AddTodo(todo);

            var items = await grain.GetAll();
            Assert.IsTrue(items.Any(it => it.Id == todo.Id));
        }
```

**Comment by ReubenBond** [comment]
@raghur it looks as though one or more of your assemblies (eg, `Orleans.TestKit`) were built against an older version of Orleans.

**Comment by dsarfati** [comment]
Yes, I should get a version using the RC out later this week.

**Comment by AdamWyzgol** [comment]
I upgrade references to RC1 #38

**Comment by dsarfati** [comment]
Orleans TestKit Nuget is now available for [2.0-RC2](https://www.nuget.org/packages/OrleansTestKit/2.0.0-RC2)
